chess/src/App.js

```diff
@@ -3,13 +3,20 @@ import "./App.css";
 import CreateAccount from "./Components/CreateAccount";
 import LogIn from "./Components/LogIn";
 import { Switch, Route, Redirect, Link } from "react-router-dom";
+import HomePage from "./Components/HomePage";
+import PlayAI from "./Components/PlayAI";
+import PlayOnline from "./Components/PlayOnline";
+import Leaderboards from "./Components/PlayOnline copy";
 
 function App() {
   return (
     <div className="App">
```

Comment:
Replace with Header component above switch tags.

Reply:
Will make header component and replace it with that now